docs/introduction/what-is-cosmos.md

```diff
@@ -1,7 +0,0 @@
-# What is Cosmos?
```

Comment:
do we need to delete this page?

Reply:
wanted to replace it with smth else but ended up not doing it. I'm adding it back. Will improve it later.